src/App.jsx

```diff
@@ -14,33 +15,94 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const generateKey = () => {
+  return `_${Math.random().toString(36).substr(2, 9)}`;
+};
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState([...goodsFromServer]);
+  const [isSortedAlphabetically, setIsSortedAlphabetically] = useState(false);
+  const [isSortedByLength, setIsSortedByLength] = useState(false);
```

Comment:
simplify it to one state
```suggestion
  const [sortValue, setSortValue] = useState('');
```
and save here the string('alphabetically' or  'byLength')